docs/tutorials/configuration.md

```diff
@@ -1,222 +1,7 @@
-# Configuration for measuring node and path latency
+# Configuration
 
-CARET can observe as following:
+Please download [the architecture file](https://raw.githubusercontent.com/tier4/CARET_demos/main/samples/end_to_end_sample/architecture.yaml) and use it for visualization.
```

Comment:
I think that it makes easear for beginners to show download commands.


```
cd ~/ros2_ws/evaluate
wget https://raw.githubusercontent.com/tier4/CARET_demos/main/samples/end_to_end_sample/architecture.yaml
```

Reply:
fixed at https://github.com/tier4/CARET_doc/pull/112/commits/cacdadc443aefa0396d51be34cef3b1551a80e41